tests/base_test_case.py

```diff
@@ -115,9 +115,9 @@ def assertRaisesRegexp(self, expected_exception, expected_regexp,
             self.fail("Expected exception " + str(expected_exception) +
                       " not raised" + msg)
 
-    def assertLazyMessage(self, assert_function, actual, expected, msg_func):
+    def assertLazyMessage(self, msg_func, assert_function, *args, **kwargs):
```

Comment:
I'm going to push it as is now.  The arguments to msg_func currently aren't all the same as those to the assert_function, so there isn't an easy 2 minute change, and I want to get the extra debugging in now in hopes of capturing something sooner.   But I agree that some sort of future clean up would be good.